bakery/README.md

```diff
@@ -0,0 +1,15 @@
+## Bakery Concourse Pipeline Generator
+
+### Setup
+- Install NodeJS and `yarn` on your machine.
+- Ensure `bakery/` is your working directory.
+- Run `yarn install` or `yarn`.
+
+### Generate a pipeline file
+- Run `yarn build [path/to/output]`
```

Comment:
It's probably useful for `yarn build` to build the `pipeline.yml` file and include it in git. At least until deployment turbulence decreases (continuing with concourse's airplane terminology)